engine/src/test/scala/ReadTsvWorkflowSpec.scala

```diff
@@ -1,12 +1,9 @@
 package cromwell
 
-import akka.testkit._
-import wdl4s.types.{WdlFileType, WdlIntegerType, WdlStringType, WdlArrayType}
-import wdl4s.values.{WdlFile, WdlInteger, WdlArray, WdlString}
-import cromwell.CromwellSpec.DockerTest
+import akka.testkit.EventFilter
 import cromwell.util.SampleWdl
-
-import scala.language.postfixOps
+import wdl4s.types.{WdlFileType, WdlStringType, WdlIntegerType, WdlArrayType}
+import wdl4s.values.{WdlFile, WdlString, WdlInteger, WdlArray}
```

Comment:
That said, if you want me to put the file back on its own little island I can certainly do that